cli/src/commit_templater.rs

```diff
@@ -1472,11 +1472,19 @@ fn builtin_commit_evolution_entry_methods<'repo>()
                 let predecessors: Vec<_> = entry.predecessors().try_collect()?;
                 let from_tree = rebase_to_dest_parent(repo, &predecessors, &entry.commit)?;
                 let to_tree = entry.commit.tree();
+                let mut copy_records = CopyRecords::default();
```

Comment:
iirc, we need copy-tracking support in rebase logic. There's a new design doc about that. 
https://docs.jj-vcs.dev/latest/design/copy-tracking/

It wouldn't make much sense to deduce renames after trees were merged/rebased wrongly.